src/applications/coronavirus-chatbot/components/ChatbotWrapper.jsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import recordEvent from '../../../platform/monitoring/record-event';
+import { GA_PREFIX } from '../utils';
+import ChatbotLoadError from './ChatbotLoadError';
+import * as Sentry from '@sentry/browser';
+import initializeChatbot from '../index';
+
+const idString = 'chatbot-wrapper-id';
+
+export class ChatbotWrapper extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      chatbotError: false,
+    };
+  }
+
+  async componentDidMount() {
+    const initialComponent = document.querySelector(`#${idString}`);
```

Comment:
If it were me, I would refactor this method to be more readable, but not a show stopper